core/store/src/db.rs

```diff
@@ -302,7 +300,8 @@ fn rocksdb_options() -> Options {
     opts.set_write_buffer_size(1024 * 1024 * 512 / 2);
     opts.set_max_bytes_for_level_base(1024 * 1024 * 512 / 2);
     opts.increase_parallelism(cmp::max(1, num_cpus::get() as i32 / 2));
-
+    opts.set_max_total_wal_size(1 * 1024 * 1024 * 1024);
```

Comment:
Can we document every option with the reason why we decided on the specific value?

Reply:
Sounds good. I'll address in a different PR this week, which will tune cache options